WeatherForecast/WeatherForecastTests/WeatherForecastTests.swift

```diff
@@ -8,25 +8,137 @@ import XCTest
 @testable import WeatherForecast
 
 class WeatherForecastTests: XCTestCase {
-
-    override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+    
+    let parsingManager = ParsingManager()
+    
+    func test_JSON파일인CurrentWeather를_디코딩했을때_id는420006353이다() {
```

Comment:
메서드명에 expectedValue 보다는 스펙을 명세할 수 있는 말로 대체해주세요 혹은 생략해도 좋을 것 같습니다
```suggestion
    func test_decode_currentWeather() {
```

Reply:
then 자리에 구체적인 하나의 값을 써주기보다 '디코딩에 성공한다'와 같은 방식으로 변경해주었습니다.